srcs/parser/check_walls.c

```diff
@@ -24,30 +24,30 @@ static void	st_find_player(t_file_data *file_data)
 	}
 }
 
-static char	**st_copy_map(t_map_data *map_data)
-{
-	int		i;
-	char	**copy;
+// static char	**st_copy_map(t_map_data *map_data)
```

Comment:
you moved this to another file right?

Reply:
yeah, I put it in utils instead, im using it to replace the spaces by 'X' for better map drawing